additional-providers/hybridauth-openstreetmap/Providers/OpenStreetMap.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/*!
+* HybridAuth
+* http://hybridauth.sourceforge.net | http://github.com/hybridauth/hybridauth
+* (c) 2017, HybridAuth authors | http://hybridauth.sourceforge.net/licenses.html 
+*/
+
+/**
+* Hybrid_Providers_OpenStreetMap (openstreetmap.org)
+*/
+class Hybrid_Providers_OpenStreetMap extends Hybrid_Provider_Model_OAuth1
```

Comment:
there are no coding standards at all. Can you fix it according to PSR2 standard?

Reply:
Thanks, I've fixed indentation, and functions calls